mobile/android/app/src/main/kotlin/app/alextran/immich/widget/configure/RandomConfigure.kt

```diff
@@ -104,6 +105,16 @@ fun RandomConfiguration(context: Context, appWidgetId: Int, glanceId: GlanceId,
     val albumEntity = availableAlbums.firstOrNull { it.id == currentAlbumId }
     selectedAlbum = albumEntity ?: availableAlbums.first()
 
+    // load filter configuration
+    val availableFilters = listOf(
+      DropdownItem("None", WidgetImageFilter.NONE.name), 
+      DropdownItem("Black & White", WidgetImageFilter.BLACK_WHITE.name)
+    )
```

Comment:
lets generate this inside of `WidgetImageFilter` so we can use `WidgetImageFilter.dropdownItems()` to retrieve this list. This makes it easier in the future if we were to add additional filters, we just modify the enum and everything else updates